RELEASE/scripts/sl_ascend/sl_cooking.ash

```diff
@@ -1995,3 +2003,26 @@ boolean sl_knapsackAutoConsume(string type, boolean simulate)
 	}
 	return true;
 }
+
+boolean sl_maximizedConsumeStuff()
+{
+	if(my_adventures() < 10)
+	{
```

Comment:
I think there's still some room to improve this "drink up to 8, then drink the rest, then knapsack-pack our stomach" logic but am declaring this Good Enough For Now.